code/modules/vote/vote_presets.dm

```diff
@@ -17,6 +17,8 @@
 /datum/vote/map
 	question = "Map Vote"
 	vote_type_text = "map"
+	vote_counting = VOTE_COUNTING_TICKET
+	show_counts = TRUE
```

Comment:
I don't like that you've dressed this up as a stealth change, and the counts are still relevant in ticket voting as it still increases the weight. 

The votes will decide though (hue).

Reply:
You are no longer incentivised to stick to the majority. So you no longer need to know who the majority is to do the right decision. So displaying the votes doesn't sabotage the vote one way or another.

In the `VOTE_COUNTING_MAJORITY`, for example, if you want "anything except Box", then you *have to* vote for whichever non-box map is currently in the lead, else your vote is completely wasted.
And this is not the problem at all with `VOTE_COUNTING_TICKET`